specs/tabs/tabs.md

```diff
@@ -0,0 +1,173 @@
+        <button class="tab" part="tab" role="tab" aria-selected="false" aria-controls="panel-2" id="tab-2" tabindex="-1">
+            Tab Two
+        </button>
+        <button class="tab" part="tab" role="tab" aria-selected="false" aria-controls="panel-3" id="tab-3" tabindex="-1">
+            Tab Three
+        </button>
+    </div>
+    <slot class="end" name="end" part="end"></slot>
+    <div class="tabpanel" part="tabpanel" id="panel-1" role="tabpanel" tabindex="0" aria-labelledby="tab-1">
+        Content of the first tab
+    </div>
+    <div class="tabpanel" part="tabpanel" id="panel-2" role="tabpanel" tabindex="0" aria-labelledby="tab-2" hidden>
+        Content of the second tab
+    </div>
+    <div class="tabpanel" part="tabpanel" id="panel-3" role="tabpanel" tabindex="0" aria-labelledby="tab-3" hidden>
+        Content of the third tab
+    </div>
+</div>
+```
+
+## Implementation
+
+*Logical grouping option 1:*
+```HTML
+<fast-tabs>
+    <fast-tab-pane tab="Tab One" key="1">
+        Content of the first tab
+    </fast-tab-pane>
+    <fast-tab-pane tab="Tab Two" key="2">
+        Content of the second tab
+    </fast-tab-pane>
+    <fast-tab-pane tab="Tab Three" key="3">
+        Content of the third tab
+    </fast-tab-pane>
+    <div slot="before-content">
+        Before content
+    </div>
+    <div slot="after-content">
+        Before content
+    </div>
+</fast-tabs>
+```
+
+*Logical grouping option 2:*
+```HTML
+<fast-tabs>
+    <fast-tab-item>
+        <fast-tab>Tab One</fast-tab>
+        <fast-tab-panel>Content of the first tab</fast-tab-panel>
+    </fast-tab-item>
+    <fast-tab-item>
+        <fast-tab>Tab Two</fast-tab>
+        <fast-tab-panel>Content of the Second tab</fast-tab-panel>
+    </fast-tab-item>
+    <fast-tab-item>
+        <fast-tab>Tab Three</fast-tab>
+        <fast-tab-panel>Content of the Third tab</fast-tab-panel>
+    </fast-tab-item>
+    <div slot="before-content">
+        Before content
+    </div>
+    <div slot="after-content">
+        Before content
+    </div>
+</fast-tabs>
+```
+
+*DOM grouping:*
+```HTML
+<fast-tabs>
+    <fast-tab-list>
+        <fast-tab>Tab One</fast-tab>
+        <fast-tab>Tab Two</fast-tab>
+        <fast-tab>Tab Three</fast-tab>
+    </fast-tab-list>
+    <fast-tab-panel>Content of the first tab</fast-tab-panel>
+    <fast-tab-panel>Content of the Second tab</fast-tab-panel>
+    <fast-tab-panel>Content of the Third tab</fast-tab-panel>
+    <div slot="before-content">
+        Before content
+    </div>
+    <div slot="after-content">
+        Before content
+    </div>
+</fast-tabs>
+```
+
+### States
+
+*Tabs* can either be controlled or uncontrolled, meaning if `activeId` is passed the app author is taking control of the selected tab. The `change` event fires differently depending on the `tab-activation` attribute, on tab focus if `tab-activation` is set to `"focus"` or on mouse click or "spacebar" press if `tab-activation` is set to `"key-press"`.
```

Comment:
I wonder if "click" would be more intuitive than "key-press" - it is shorter and is a "click" event can be fired from keystrokes like enter and spacebar (depending on the element)

Reply:
Something weird happened here, this spec is in a separate review. I'll remove this from this PR. If you could review that one here: https://github.com/microsoft/fast-dna/pull/2857